src/engine/ScreenFilter.tscn

```diff
@@ -0,0 +1,12 @@
+[gd_scene load_steps=3 format=2]
+
+[ext_resource path="res://src/engine/ScreenFilter.cs" type="Script" id=1]
+
+[sub_resource type="ShaderMaterial" id=2]
```

Comment:
In this case it is likely not a problem but for good practice this should be marked as scene unique resource as I assume you modify the shader from the code and that will absolutely break in the case where multiple instances of this scene were loaded. Which isn't the case here but using the proper way to do here might help someone realize that that's the way to do it.

Reply:
I don't know how to do this, in this context

Reply:
![kuva](https://user-images.githubusercontent.com/3796411/235657558-f1cb81f4-a60d-48c5-bbfb-da0bf2e32e0a.png)

check that box in the editor for that material in the scene.